otx/algorithms/action/configs/classification/x3d/model.py

```diff
@@ -0,0 +1,128 @@
+"""Model configuration of X3D model for Action Classification Task."""
+
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+# pylint: disable=invalid-name
+
+num_classes = 400
+num_samples = 12
+model = dict(
+    type="Recognizer3D",
+    backbone=dict(type="X3D", gamma_w=1, gamma_b=2.25, gamma_d=2.2),
+    cls_head=dict(
+        type="X3DHead", in_channels=432, num_classes=num_classes, spatial_type="avg", dropout_ratio=0.5, fc1_bias=False
+    ),
+    # model training and testing settings
+    train_cfg=None,
+    test_cfg=dict(average_clips="prob"),
+)
+
+
+# dataset settings
+seed = 2
+dataset_type = "RawframeDataset"
+
+img_norm_cfg = dict(mean=[0.0, 0.0, 0.0], std=[255.0, 255.0, 255.0], to_bgr=False)
+
+clip_len = 8
+frame_interval = 4
+train_pipeline = [
+    dict(type="SampleFrames", clip_len=clip_len, frame_interval=frame_interval, num_clips=1),
+    dict(type="RawFrameDecode"),
+    dict(type="Resize", scale=(-1, 256)),
+    dict(type="RandomResizedCrop"),
+    dict(type="Resize", scale=(224, 224), keep_ratio=False),
+    dict(type="Flip", flip_ratio=0.5),
+    dict(type="Normalize", **img_norm_cfg),
+    dict(type="FormatShape", input_format="NCTHW"),
+    dict(type="Collect", keys=["imgs", "label"], meta_keys=[]),
+    dict(type="ToTensor", keys=["imgs", "label"]),
+]
+
+val_pipeline = [
+    dict(type="SampleFrames", clip_len=clip_len, frame_interval=frame_interval, num_clips=1, test_mode=True),
+    dict(type="RawFrameDecode"),
+    dict(type="Resize", scale=(-1, 256)),
+    dict(type="CenterCrop", crop_size=224),
+    dict(type="Normalize", **img_norm_cfg),
+    dict(type="FormatShape", input_format="NCTHW"),
+    dict(type="Collect", keys=["imgs", "label"], meta_keys=[]),
+    dict(type="ToTensor", keys=["imgs"]),
+]
+# TODO Delete label in meta key in test pipeline
+test_pipeline = [
+    dict(type="SampleFrames", clip_len=clip_len, frame_interval=frame_interval, num_clips=1, test_mode=True),
+    dict(type="RawFrameDecode"),
+    dict(type="Resize", scale=(-1, 256)),
+    dict(type="CenterCrop", crop_size=224),
+    dict(type="Normalize", **img_norm_cfg),
+    dict(type="FormatShape", input_format="NCTHW"),
+    dict(type="Collect", keys=["imgs"], meta_keys=[]),
+    dict(type="ToTensor", keys=["imgs"]),
+]
+
+data = dict(
+    videos_per_gpu=10,
+    workers_per_gpu=0,
```

Comment:
Is there any reason to set this value as '0'? for the stability issue?

Reply:
For debug issue, pdb do not work in multi-processing. It can be changed after we add our PoC candidates